### PR TITLE
Addition of two new types of HWS (HWS_SWTM_WDPC_BYPVPM and HWS_SWTC_WDPC_BYPVPM_HWVM)

### PR DESCRIPTION
**Pull request by ghairapetian**

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/HWS.yaml [hunk 1]
```diff
@@ -231,6 +231,27 @@ HWS_SWTC_BYPVPM_WDPC_WFRM:
   - WDPC
   - WFRC
 
+HWS_SWTM_WDPC_BYPVPM:
+  id: "12025993778390106113"
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/HWS.yaml:235]: Please remove ID.

File: ontology/yaml/resources/HVAC/entity_types/HWS.yaml [hunk 2]
```diff
@@ -231,6 +231,27 @@ HWS_SWTC_BYPVPM_WDPC_WFRM:
...
+  description: "Heating water system with supply temperature monitoring, differential pressure control and bypass valve control."
+  is_canonical: true
+  implements:
+  - HWS
+  - SWTM
+  - WDPC
+  - BYPVPM
+
+HWS_SWTC_WDPC_BYPVPM_HWVM:
+  id: "12025993778390106114"
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/HWS.yaml:245]: Please remove ID.

**Review by tasodorff — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/HWS.yaml (hunk 2 above)
Comment (ghairapetian) [reply ontology/yaml/resources/HVAC/entity_types/HWS.yaml]: Done

File: ontology/yaml/resources/HVAC/entity_types/HWS.yaml (hunk 1 above)
Comment (ghairapetian) [reply ontology/yaml/resources/HVAC/entity_types/HWS.yaml]: Done

**Review by tasodorff — APPROVED** [review]
All new combinations of existing abstract types. Looks good to me.

**Pull request merged**